File: hw/collahuazo/u3/hw24CompositePaterns/CompositeEmployee/src/main/java/ec/edu/espe/composite/model/Clerk.java
package ec.edu.espe.composite.model;

/**
 *
 * @author Brandon Collahuazo,CodeBros,@ESPE
 */
public class Clerk extends Employee {

    public Clerk(String aName) {
        this();
        name = aName;
    }

    public Clerk() {
        title = "Clerk";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}

File: hw/nase/u3/hw24CompositePattern/CompossitePattern.javaScript/model/Clerk.js

import { Employee } from './Employee.js';

//Jennyfer Nase

export class Clerk extends Employee {
    constructor(aName = null) {
        super();
        this.title = "Clerk";
        if (aName !== null) {
            this.name = aName;
        }
    }

    stateName() {
        super.stateName();
    }
}